tests/unit/test-class-wc-payments-token-service.php

```diff
@@ -157,7 +157,7 @@ public function test_woocommerce_payment_token_deleted() {
 
 	public function test_woocommerce_payment_token_deleted_other_gateway() {
 		$this->mock_api_client
-			->expects( $this->never() )
+			->expects( $this->once() )
```

Comment:
It sounds like this unit test was written to make sure that deleting a payment token from another gateway doesn't happen. Setting this to `once()` seems like it kinda breaks the initial intention of the test. 
I wonder if we can write perhaps a more specific test for split UPE gateways? 
For instance, if we delete a SEPA gateway we test that only a SEPA token type is deleted(and only once)? 🤔